ssg/yaml.py

```diff
@@ -28,7 +28,7 @@ def _save_rename(result, stem, prefix):
     result["{0}_{1}".format(prefix, stem)] = stem
 
 
-def _open_yaml(stream, original_file=None):
+def _open_yaml(stream, original_file=None, substitutions_dict={}):
```

Comment:
I think the one new scrutinizer issue here is a non-issue.

Reply:
In this case it's not an issue but mutable default args are a huge issue in Python. It's super confusing and hard to debug and not even reentrant. I can change it to substitutions_dict=None and then instantiate a dict if it's None inside the function.

Again, in this specific case it's not an issue because we never modify it but if anyone adds a code that adds something to that dict we will be surprised about the results :-D